daemon/stats/collector_unix.go

```diff
@@ -16,8 +16,11 @@ const (
 	// on Linux it's a constant which is safe to be hard coded,
 	// so we can avoid using cgo here. For details, see:
 	// https://github.com/containerd/cgroups/pull/12
-	clockTicksPerSecond  = 100
-	nanoSecondsPerSecond = 1e9
+	clockTicksPerSecond  uint64 = 100
+	nanoSecondsPerSecond        = 1e9
+	nanoSecondsPerTick          = nanoSecondsPerSecond / clockTicksPerSecond
+	// 18446744073709551615 is the max value for uint64 (2^64)
+	maxClockTicks = 18446744073709551615 / nanoSecondsPerTick
```

Comment:
Perhaps use `math. MaxUint64 ` for this (https://golang.org/pkg/math/#pkg-constants), then there's no need to add the comment

```suggestion
	maxClockTicks = math.MaxUint64 / nanoSecondsPerTick
```

Reply:
Was looking for this and couldn't seem to find it. Thanks for the link! Fixed

Reply:
I had to search as well 😂 I recalled there was a `const` "somewhere", but couldn't recall where it was 👍